server/hack/deploy.sh

```diff
@@ -14,12 +14,14 @@ SERVER_LOG_LEVEL=${SERVER_LOG_LEVEL:-0}
 #
 # if the namespace doesn't exist, then head will return a failure (since it has
 # an empty input).  Since we manually check the error case, disable pipefail
-set +o pipefail
-toolchain_host=$(${KUBECLI} get namespaces -o name | grep toolchain-host | cut -d'/' -f2 | head -n 1)
-if [[ -z "${toolchain_host}" ]]; then
-    toolchain_host="toolchain-host_operator"
+if [[ -z "${TOOLCHAIN_HOST}" ]]; then
+    set +o pipefail
+    TOOLCHAIN_HOST=$(${KUBECLI} get namespaces -o name | grep toolchain-host | cut -d'/' -f2 | head -n 1)
```

Comment:
The `grep` should already filter out the header here, but to be highly expressive and simplify this online we may use `--no-headers`
```suggestion
    TOOLCHAIN_HOST=$(${KUBECLI} get namespaces -o name --no-headers | grep toolchain-host | cut -d'/' -f2 )
```

Reply:
Like `toolchain-host_operator`, this is what was getting set prior to this PR.  I don't hit this codepath during releases, but I can still change it if you'd like.

Reply:
Good either way for me. Either we commit the suggestion here, in a separate PR, or just ignore. It's just a cosmetic thing.

Reply:
I'll ignore it for now, let's fix it in a separate PR.  It's unrelated to the stuff I want to do in this PR.